.github/workflows/ci.yaml

```diff
@@ -10,14 +10,14 @@ jobs:
   build:
     runs-on: ubuntu-latest
     env:
-      python_version: '3.9.x'
+      python_version: "3.9.x"
       IMAGE: praekeltfoundation/rapidpro
       REGISTRY_USER: rapidproautomation
-      REPO: praekeltfoundation/rapidpro
+      REPO: rapidpro/rapidpro
```

Comment:
@KaitCrawford @DevChima @codiebeulaine  FYI we're changing our docker builds to use rapidpro/rapidpro directly so that we don't have to keep updating our fork. We don't have any custom things in there anymore.